### DISCUSSION TITLE
Intercepting commands in code server

### DISCUSSION THREAD

**Discussion by singhdevhub-lovepreet** [coder/code-server#6849] [Q&A]
If I am running code server in a docker env and wanted to intercept the commands which someone gives in the terminal. 

For ex: I want to preprocess some commands that someone enters in the terminal of code-server, How one can do that

**Comment by code-asher** [comment]
From code-server's point of view, there is no way to do this yet I believe but I think it will be possible through an extension once this goes through: https://github.com/microsoft/vscode/issues/145234
